croncheck.php

```diff
@@ -117,6 +119,13 @@
     header('Expires: Tue, 04 Sep 2012 05:32:29 GMT');
 }
 
+// IP Locking, check for remote IP in validated list, make sure not run from CLI, if not, exit    $allowedips = get_config('tool_heartbeat','ipconfig');
```

Comment:
I'd refactor this whole chunk of code into iplock so this is just a one liner like:

check_ip();

Reply:
Currently unable to put the get_config inside of IP Lock, causes a bug with cron on the croncheck web execution. Tried it for a while, couldn't get it to play nice